core/src/test/java/io/questdb/network/NetTest.java

```diff
@@ -73,6 +73,20 @@ public void testBindAndListenUdpToLocalhost() {
         }
     }
 
+    @Test
+    public void testGetAddrInfoConnect() {
+        NetworkFacade nf = NetworkFacadeImpl.INSTANCE;
+            final long lpAddrInfo = nf.getAddrInfo("questdb.io", 443);
```

Comment:
odd formatting

Reply:
fixed in https://github.com/questdb/questdb/pull/2239/commits/6d7821f107680b9e78313e6509f5116e81a081f9